revenuecat_examples/purchase_tester/lib/src/paywall_footer_screen.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:purchases_ui_flutter/purchases_ui_flutter.dart';
+import 'package:purchases_flutter/purchases_flutter.dart';
+import 'package:flutter/material.dart';
+
+class PaywallFooterScreen extends StatefulWidget {
+  final Offering? offering;
+
+  const PaywallFooterScreen({Key? key, this.offering}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _PaywallFooterScreenState();
+}
+
+class _PaywallFooterScreenState extends State<PaywallFooterScreen> {
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea( // Wrap your body content with SafeArea
+        child: Center(
+          child: PaywallFooterView(
+            offering: widget.offering,
+            content: Container(
+              color: Colors.blue.withAlpha(80),
+              child: SingleChildScrollView(
+                padding: const EdgeInsets.only(bottom: PaywallFooterView.roundedCornerRadius),
```

Comment:
we are adding the padding ourselves in Android and iOS right? I think we should match that behavior

Reply:
Well, in iOS, the system for the footer is different (it's not a container). In android it is and what we do is passing the padding as part of the function that accepts the children composables... I guess we could make the child here a function instead, that accepts a padding and returns a widget... I mostly wanted to do it this way to keep it in sync with the way it works in the native flutter components... But it might be cleaner yeah...

Reply:
I made this change in https://github.com/RevenueCat/purchases-flutter/pull/996/commits/0624e6dbb251f66c1b9b54e2e3a6cc0f289cc214. Lmk what you think @vegaro